File: GooglePay/src/main/java/com/braintreepayments/api/GooglePayLauncherCallback.java
package com.braintreepayments.api;

/**
 * Callback used to instantiate a {@link GooglePayLauncher} to handle Activity results from the
 * Google Pay payment flow
 */
public interface GooglePayLauncherCallback {

    void onResult(GooglePayPaymentAuthResult googlePayPaymentAuthResult);
}

File: GooglePay/src/main/java/com/braintreepayments/api/GooglePayLauncherCallback.kt
package com.braintreepayments.api

/**
 * Callback used to instantiate a [GooglePayLauncher] to handle Activity results from the Google Pay
 * payment flow
 */
fun interface GooglePayLauncherCallback {

    fun onGooglePayLauncherResult(googlePayPaymentAuthResult: GooglePayPaymentAuthResult?)
}
